colcon_core/argument_default.py

```diff
@@ -54,6 +54,23 @@ def wrap_default_value(value):
     return value
 
 
+def unwrap_default_value(value):
+    """
+    Unwrap a default value to the native type.
+
+    :param value: The wrapped default value
+    :returns: The unwrapped value
+    :raises ValueError: if the value is not a wrapped default value
+    """
+    global _types
+    if not is_default_value(value):
+        raise ValueError(
+            'the passed value is not a wrapped default value: ' +
+            str(type(value)))
+    type_ = [k for k, v in _types.items() if v == type(value)][0]
```

Comment:
Why do we need to use list comprehension instead of something like the following?

```python
return _type[type(value)](value)
```

Reply:
The comprehension is getting the *key* for `typ(value)`. Afaik there is no way to retrieve that from a `dict` without iterating it.

Reply:
ah, I missed that it was getting the key.